content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,251 @@
+To display the data in a pivot table, click on the :guilabel:`Pivot` icon located in the far-right
+side of the top menu bar.
 
-.. image:: reporting/measures-bottom.png
+The default information displayed includes the number of payslips (:guilabel:`# Payslip`), the
+:guilabel:`Net Wage`, the :guilabel:`Gross Wage`, the number of :guilabel:`Days of Paid Time Off`,
+and the number of :guilabel:`Days of Unpaid Time Off`. The information is organized by department.
+
+To display more information on the report, click the :guilabel:`Measures` button to reveal a
+drop-down menu. Then, click on any other metric to display it.
+
+.. image:: reporting/pivot.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: Pivot table view with the various metrics called out.
 
 To sort the entries by a specific column, such as :guilabel:`Net Wage`, click on the column name
 twice. The first click selects the column, and the second click sorts the information in descending
 order.
 
-.. image:: reporting/net-wage.png
-   :align: center
-   :alt: To sort information by descending order, click the column head twice.
-
-To export the data in an XLSX format, click the :guilabel:`Download xlsx` icon. The information will
-be downloaded into an Excel spreadsheet.
+To export the data in an XLSX format, click the :guilabel:`Download xlsx` icon. The information is
+then downloaded into a spreadsheet.
 
-.. image:: reporting/export.png
+.. image:: reporting/xlsx.png
    :align: center
-   :alt: Download an Excel spreadsheet of the data by clicking the download button.
+   :alt: The Download button highlighted.
 
-The data can be inserted into a spreadsheet by clicking the :guilabel:`Insert in Spreadsheet`
-button. A pop-up appears asking which spreadsheet to place the information in. Select an existing
-spreadsheet or enter the name for a new spreadsheet. Click the :guilabel:`Confirm` button to move to
-a spreadsheet view with all the information populated.
+Any report can be inserted into a spreadsheet by clicking the :guilabel:`Insert in Spreadsheet`
+button. A :guilabel:`Select a spreadsheet to insert your (type of report)` pop-up window appears,
+asking which spreadsheet to place the information in. Select an existing spreadsheet or dashboard,
+or select a new :guilabel:`Blank spreadsheet`. Click the :guilabel:`Confirm` button to move to a
+spreadsheet view with the report added to it.
 
 .. image:: reporting/spreadsheet.png
    :align: center
-   :alt: Send the data to a spreadsheet.
-
-Once in the spreadsheet view, click on :guilabel:`File`, then select :guilabel:`Save` to save the
-data, or :guilabel:`Save as Template` to save the setting as a template to use in the future.
+   :alt: The view of data sent to a spreadsheet.
 
 .. note::
-   The *Documents* application must be installed in order to use the :guilabel:`Insert in
-   Spreadsheet` option.
-
+   The *Documents* application must be installed to use the :guilabel:`Insert in Spreadsheet`
+   option.
```

Comment:
As per our discussion- this doesn't seem to be true. You can still use the "Insert in Spreadsheet" option. The spreadsheet just appears in the Dashboards app.